make-checkout

```diff
@@ -70,11 +70,24 @@ def main():
     execute("git", "checkout", "--detach", opts.revision, cwd=TARGET_DIR, error_on_fail=False)
 
     # get bots/
-    if api.repo != "cockpit-project/cockpit" or (opts.base and opts.base != "master"):
-        execute("git", "remote", "add", "bots_origin", "https://github.com/cockpit-project/cockpit", cwd=TARGET_DIR)
-        execute("git", "fetch", "--depth=1", "bots_origin", opts.bots_ref or "master", cwd=TARGET_DIR)
-        bots_ref = execute("git", "rev-parse", "FETCH_HEAD", cwd=TARGET_DIR).strip()
-        execute("git", "checkout", "--force", bots_ref or "bots_origin/master", "--", "bots/", cwd=TARGET_DIR, error_on_fail=False)
+    bots_clone_dir = os.path.join(TARGET_DIR, "bots")
+
+    # HACK
+    # As long as bots/ is duplicated in cockpit, keep the conditional section here
+    if api.repo == "cockpit-project/cockpit":
+        if opts.base and opts.base != "master":
+            execute("git", "remote", "add", "bots_origin", "https://github.com/cockpit-project/bots", cwd=TARGET_DIR)
```

Comment:
As long as the One True Bots directory is still in cockpit, let's please use that, not cockpit-project/bots. We can flip this over once we declare cockpit-project/bots as The Truth.

For PRs against cockpit-project/cockpit master, this plus the special case above will essentially mean to remove bots/ and replace it with an identical copy. This should be fine, and the extra overhead is only temporary.